### ISSUE TITLE
Adding our African Stopwords

### ISSUE THREAD

**Issue by chrisemezue** [nltk/nltk_data#201]
Original post [here](https://github.com/nltk/nltk/issues/3141). 
**TL;DR:** The African stopwords project is an ever-ongoing project to curate trusted stopwords for African languages. I am proposing a collaboration between NLTK.org, [Masakhane](https://www.masakhane.io/) and [Lanfrica ](https://lanfrica.com/) to enable the inclusion of African languages in the NTLK, starting with stopwords.

- The homebase and metadata of our African stopwords is [here](https://github.com/masakhane-io/masakhanePreprocessor/tree/main/african-stopwords
).

- The stopwords themselves can be found [here](https://github.com/masakhane-io/masakhanePreprocessor/tree/main/african-stopwords/languages
).

Let me know if there is anything more needed from our side. 

_Apologies for taking long on this._
